### PR TITLE
Lint models/admin.go

### PR DESCRIPTION
**Pull request by Bwko**

### CODE REVIEW

**Comment by strk** [comment]
LGTM

File: models/admin.go [hunk 1]
```diff
@@ -18,25 +18,27 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 )
 
-type NoticeType int
+type noticeType int
```
Comment (andreynering) [inline models/admin.go]: I'd keep this public, since it's used on a public field of a public struct below (`Notice`).

:+1:  for the other changes.

**Comment by Bwko** [comment]
@andreynering  Updated

File: models/admin.go [hunk 2]
```diff
@@ -18,10 +18,11 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 )
 
+//NoticeType describes the notice type
 type NoticeType int
 
 const (
-	NoticeRepository NoticeType = iota + 1
+	noticeRepository NoticeType = iota + 1
```
Comment (andreynering) [inline models/admin.go]: Sorry for missing, but this should be public, too.

**Comment by Bwko** [comment]
@andreynering my bad, should have seen that one.

**Comment by andreynering** [comment]
LGTM

**Comment by tboerger** [comment]
LGTM

**Comment by lunny** [comment]
resolved #70

**Pull request merged**